tests/docker-build-httpd/main.yml

```diff
@@ -0,0 +1,81 @@
+---
+# vim: set ft=ansible:
+#
+# !!!NOTE!!! This playbook was tested using Ansible 2.1; it is recommended
+# that the same version is used.
+#
+# This playbook aims to verify that the changes to the Atomic Host platform
+# do not disrupt the ability for users to build/run containers built on top
+# of various base images.  The base images from Docker Hub were selected
+# based on 'popularity' (aka number of pulls).
+#
+# Each base image is used to build an httpd container, which is then run,
+# and the content returned by the container is verified to be correct.
+# Afterwards, the container and build image are removed from the system.
+#
+- name: Docker build httpd test
+  hosts: all
+  become: yes
+
+  # This is a hack.  Using the 'redhat_subscription' role as an 'include' was
+  # throwing an error about the 'subscription_file' variable not being
+  # defined, despite there being 'roles/redhat_subscription/vars/main.yaml'
+  # with the variable defined there.  This kind of short-circuits the role
+  # by providing the value at the playbook level.
+  vars:
+    subscription_file: 'roles/redhat_subscription/files/subscription_data.csv'
```

Comment:
Does it make sense to have multiple plays in this scenario?  Maybe a 'setup' play that will do the subscription with post_task to manage the variables pull the images and then have a separate play that runs the test.  I'm not sure if this makes sense in this situation because way you have it now is much cleaner.